test/kata2.booleanToWord.test.js

```diff
@@ -2,4 +2,17 @@ const { booleanToWord } = require("../src");
 
 describe("booleanToWord", () => {
   // how do we create specs again???
+  it('returns true for Yes', ()=>{
```

Comment:
I think you've misunderstood what this kata is supposed to do. Looking at the function it accepts a single parameter which is a boolean, and the function should return 'yes' if its `true`, and 'no' if its `false`. Here your tests dont take any parameters so you should rewrite this to take that into consideration (you only need two test cases, one for passing in `true` and one for `false`)